docs/pcp-users-and-administrators-guide.rst

```diff
@@ -0,0 +1,7051 @@
+   Allows retrieval of information from the PMCD. This may be
+   information about a metric (such as a description, instance domain,
+   or help text) or an actual value for a metric.
+
+``store``
+   Allows the PMCD to store metric values in PMDAs that permit store
+   operations. Be cautious in allowing this operation, because it may be
+   a security opening in large networks, although the PMDAs shipped with
+   the PCP package typically reject store operations, except for
+   selected performance metrics where the effect is benign.
+
+For example, here is a sample access control portion of a
+``${PCP_PMCDCONF_PATH}`` file:
+
+::
+
+   allow hosts babylon, moomba : all ; 
+   disallow user sam : all ;
+   allow group dev : fetch ; 
+   allow hosts 192.127.4.* : fetch ; 
+   disallow host gate-inet : store ; 
+
+Complete information on access control syntax rules in the ``pmcd.conf``
+file can be found in the ``pmcd(1)`` man page.
+
+.. _LE43202-PARENT:
+
+Managing Optional PMDAs
+-----------------------
+
+Performance Metrics Domain Agents
+(PMDAs) shipped with Performance Co-Pilot (PCP) are designed to be
+installed and activated on every collector host, for example, ``linux``,
+``windows``, ``darwin``, ``pmcd``, and ``process`` PMDAs.
+
+Other PMDAs are designed for optional activation and require some user
+action to make them operational. In some cases these PMDAs expect local
+site customization to reflect the operational environment, the system
+configuration, or the production workload. This customization is
+typically supported by interactive installation scripts for each PMDA.
+
+Each PMDA has its own directory located below ``${PCP_PMDAS_DIR}``. Each
+directory contains a ``Remove`` script to unconfigure the PMDA, remove
+the associated metrics from the PMNS, and restart the ``pmcd`` daemon;
+and an ``Install`` script to install the PMDA, update the PMNS, and
+restart the PMCD daemon.
+
+As a shortcut mechanism to support automated PMDA installation, a file
+named ``.NeedInstall`` can be created in a PMDA directory below
+``${PCP_PMDAS_DIR}``. The next restart of PCP services will invoke that
+PMDAs installation automatically, with default options taken.
+
+.. _LE31599-PARENT:
+
+PMDA Installation on a PCP Collector Host
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To install a PMDA you must perform a collector
+installation for each host on which the PMDA is required to export
+performance metrics. PCP provides a distributed metric namespace (PMNS)
+and metadata, so it is not necessary to install PMDAs (with their
+associated PMNS) on PCP monitor hosts.
+
+You need to update the PMNS, configure the PMDA, and
+notify PMCD. The ``Install`` script for each PMDA automates these
+operations, as follows:
+
+1. Log in as ``root`` (the superuser).
+
+2. Change to the PMDA's directory as shown in the following example:
+
+   ::
+
+      cd ${PCP_PMDAS_DIR}/cisco
+
+3. PMDPerformance Metrics DomainPMDIn the unlikely event that you wish
```

Comment:
Something's off here, where is `PMDPerformance Metrics DomainPMD` coming from? Did you write a script to automatically convert the xml to rst (https://github.com/performancecopilot/pcp/blob/master/books/PCP_UAG/pcp-users-and-administrators-guide.xml#L1847)? (Good idea btw :+1:)

Reply:
Oops, it seems that there is a conversion error here. Actually, a converting tool was used for this task. I would need some extra time to review and improve the docs. Thanks for the notice!